channels-and-providers/email/maqsam.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Maqsam'
+description: 'Learn how to use the Maqsam provider to send sms notifications using Novu'
+---
+
+[Maqsam](https://maqsam.com/) is the friendliest cloud telephony solution for SMEs.
```

Comment:
Please rephrase to:

You can use the [Maqsam](https://maqsam.com/) provider to send SMS messages to your customers using the Novu platform with a single API to create multi-channel experiences.